yarn_api_client/application_master.py

```diff
@@ -15,17 +15,19 @@ class ApplicationMaster(BaseYarnAPI):
     If `address` argument is `None` client will try to extract `address` and
     `port` from Hadoop configuration files.
 
-    :param str address: Proxy HTTP address
-    :param int port: Proxy HTTP port
+    :param str url: ApplicationMaster HTTP(S) address
     :param int timeout: API connection timeout in seconds
-    :param boolean kerberos_enabled: Flag identifying is Kerberos Security has been enabled for YARN
+    :param AuthBase auth: Auth to use for requests
+    :param boolean verify: Either a boolean, in which case it controls whether
+    we verify the server's TLS certificate, or a string, in which case it must
+    be a path to a CA bundle to use. Defaults to ``True``
     """
-    def __init__(self, address=None, port=8088, timeout=30, kerberos_enabled=False):
-        if address is None:
+    def __init__(self, url=None, timeout=30, auth=None, verify=True):
```

Comment:
could you please rename url to service_endpoint

Reply:
Addressed